src/main/java/md/maib/config/JdbcConfig.java

```diff
@@ -29,7 +29,6 @@ public static Connection getConnection() throws SQLException, ClassNotFoundExcep
         if (instance == null) {
```

Comment:
Remove the declaration of thrown exception 'java.lang.ClassNotFoundException', as it cannot be thrown from method's body.